build_tools/docker/manage_images.py

```diff
@@ -172,7 +175,8 @@ def stream_command(command, dry_run=False):
 def check_stream_command(command, dry_run=False):
   exit_code = stream_command(command, dry_run=dry_run)
   if exit_code != 0:
-    print(f'Command failed with exit code {exit_code}: `{" ".join(command)}`')
+    colored_print(
+        f'Command failed with exit code {exit_code}: `{" ".join(command)}`')
```

Comment:
I would have different colors for different situations. This is an error and many of these are info. At which point we should probably be using a logging library instead of print.

Reply:
I think that being able to distinguish between what info we print and the thousands of lines that docker prints is a net positive regardless of the particular color that we use.

The python logging module appears to only actually provide output for warnings or higher, and the output it provides is uncolored, so it won't be any easier to notice than just printing without using find.

```
>>> logging.info('Hey')
>>> logging.warning('Hey')
WARNING:root:Hey
```

Reply:
https://pypi.org/project/coloredlogs/ looks nice

Reply:
Are we okay requiring people to install packages to run our scripts?

Reply:
Yeah -1 on adding dependencies just for colored logs. We also have some stuff that depends on absl, but so far we've only used that where it's already required for TF and avoided it for standalone scripts

Reply:
FWIW, `coloredlogs` does "work" in PowerShell on Windows for me (note the blue text on a blue default background :P)

![image](https://user-images.githubusercontent.com/4010439/95604113-126dc080-0a0c-11eb-9b27-e1fdf8a04be2.png)

We can assume anyone running these scripts has a modern terminal with support for colors. We have less influence on how logs are displayed on tools (Travis, GitHub, etc.) though.

Agreed about a dependency just for colored logs being a bit too much, as much as I like pretty colors. What about trying to import and just not enabling colors if not available?

Reply:
These scripts aren't run on any CI though. I like the idea of an optional dependency that we pull in if it's available.